Src/BootCamp.Chapter/Computer/Body.cs

```diff
@@ -6,5 +6,11 @@ namespace BootCamp.Chapter.Computer
 {
     public class Body
     {
+        private string _body;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make '_body' 'readonly'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5124353)

Reply:
done